2.4/README.md

```diff
@@ -25,6 +25,9 @@ The following environment variables influence the MongoDB configuration file. Th
 |  `MONGODB_NOPREALLOC` | Disable data file preallocation.                                          |  true
 |  `MONGODB_SMALLFILES` | Set MongoDB to use a smaller default data file size.                      |  true
 |  `MONGODB_QUIET`      | Runs MongoDB in a quiet mode that attempts to limit the amount of output. |  true
+|  `MONGODB_TEXT_SEARCH_ENABLED` | Enables the [text search](https://docs.mongodb.org/v2.4/core/index-text/#text-search-text-command) feature.  |  false
+
+**Notice: The `MONGODB_TEXT_SEARCH_ENABLED` option is deprecated in MongoDB 2.6.**
```

Comment:
@bparees It is deprecated in MongoDB https://docs.mongodb.org/v2.6/reference/parameters/#param.textSearchEnabled and enabled by default.

It makes sense to enable textSearch. Does someone requested to be able to disable it?
MongoDB 2.4 image may be EOLed (https://github.com/openshift/mongodb/pull/127#issuecomment-185800731) and newer version have it enable by default.

IMPOV having this option only for consistency with 3 years old version which we don't know how long it will be supported by OpenShift does not worth adding more and more options to images and increasing complexity of images.